plugin/syncableAttribute.go

```diff
@@ -0,0 +1,170 @@
+package plugin
+
+import (
+	"fmt"
+	"time"
+)
+
+// AtimeAttr represents the Atime attribute in plugin.EntryAttributes
+func AtimeAttr() SyncableAttribute {
```

Comment:
I think these should all return pointers to the type so that every Sync call shares a pointer to a single instance rather than a copy of the SyncableAttribute.

Reply:
Sure, I can do that.